java/mtsj/core/src/main/java/io/oasp/application/mtsj/bookingmanagement/service/impl/rest/BookingmanagementRestServiceImpl.java

```diff
@@ -13,11 +14,13 @@
 import io.oasp.application.mtsj.bookingmanagement.logic.api.to.TableSearchCriteriaTo;
 import io.oasp.application.mtsj.bookingmanagement.service.api.rest.BookingmanagementRestService;
 import io.oasp.module.jpa.common.api.to.PaginatedListTo;
+import org.springframework.validation.annotation.Validated;
 
 /**
  * The service implementation for REST calls in order to execute the logic of component {@link Bookingmanagement}.
  */
 @Named("BookingmanagementRestService")
+@Validated
```

Comment:
I am on it to check the necessity of this annotation.
I contacted @hohwille as I am afraid, that we call validation twice. Since jax-rs 2.0 already does bean validation by itself, we would even run bean validation twice---once by jax-rs and once by spring framework.

Reply:
IMHO the validations should be added too to the SearchCriteria objects and the @Valid on the REST services where these object are used as parameters